src/index.ts

```diff
@@ -14,16 +15,57 @@ import {
   Server,
   SnapshotsFetcherComponents,
 } from './types'
-import { coerceEntityDeployment, contentServerMetricLabels, sleep, streamToBuffer } from './utils'
+import { contentServerMetricLabels, sleep, streamToBuffer } from './utils'
+import { PointerChangesSyncDeployment, SnapshotSyncDeployment, SyncDeployment } from '@dcl/schemas'
 
 export { metricsDefinitions } from './metrics'
 export { IDeployerComponent } from './types'
+export { createProcessedSnapshotsComponent } from './processed-snapshots'
 
 if (parseInt(process.version.split('.')[0]) < 16) {
   const { name } = require('../package.json')
   throw new Error(`In order to work, the package ${name} needs to run in Node v16 or newer to handle streams properly.`)
 }
 
+async function downloadProfileAvatars(
```

Comment:
It is really hard to tell what this function does if we don't read to the name `downloadProfileAvatars` we may create an issue for it to tackle it later?

Reply:
https://github.com/decentraland/snapshots-fetcher/issues